segmentation/include/pcl/segmentation/impl/extract_labeled_clusters.hpp

```diff
@@ -40,25 +40,44 @@
 #include <pcl/segmentation/extract_labeled_clusters.h>
 
 //////////////////////////////////////////////////////////////////////////////////////////////
-template <typename PointT> void
-pcl::extractLabeledEuclideanClusters (const PointCloud<PointT> &cloud,
-                                      const typename search::Search<PointT>::Ptr &tree,
-                                      float tolerance,
-                                      std::vector<std::vector<PointIndices> > &labeled_clusters,
-                                      unsigned int min_pts_per_cluster,
-                                      unsigned int max_pts_per_cluster,
-                                      unsigned int)
+template <typename PointT>
+void
+pcl::extractLabeledEuclideanClusters(
+    const PointCloud<PointT>& cloud,
+    const typename search::Search<PointT>::Ptr& tree,
+    float tolerance,
+    std::vector<std::vector<PointIndices>>& labeled_clusters,
+    unsigned int min_pts_per_cluster,
+    unsigned int max_pts_per_cluster,
+    unsigned int)
 {
-  if (tree->getInputCloud ()->size () != cloud.size ())
-  {
+  pcl::extractLabeledEuclideanClusters<PointT>(cloud,
+                                               tree,
+                                               tolerance,
+                                               labeled_clusters,
+                                               min_pts_per_cluster,
+                                               max_pts_per_cluster);
+}
+
+template <typename PointT>
+void
+pcl::extractLabeledEuclideanClusters(
+    const PointCloud<PointT>& cloud,
+    const typename search::Search<PointT>::Ptr& tree,
+    float tolerance,
+    std::vector<std::vector<PointIndices>>& labeled_clusters,
+    unsigned int min_pts_per_cluster,
+    unsigned int max_pts_per_cluster)
+{
+  if (tree->getInputCloud()->points.size() != cloud.points.size()) {
```

Comment:
The `points` should be avoided, no?

Reply:
The parent commit was quite old and when I rebased, these changes passed through. Reverting them